tests/tests_lite/utilities/test_distributed.py

```diff
@@ -1,9 +1,29 @@
+from functools import partial
+
 import pytest
 import torch
-from tests_lite.helpers.runif import RunIf
 
+from lightning_lite.plugins.environments import LightningEnvironment
 from lightning_lite.utilities.distributed import gather_all_tensors
-from tests_pytorch.core.test_results import spawn_launch
+from pytorch_lightning.accelerators import CPUAccelerator, CUDAAccelerator, MPSAccelerator
```

Comment:
is this intentional?

Reply:
Yes. We don't really want `tests_lite` to import from `tests_pytorch`. It was done out of laziness. This also explains why you had to install torchmetrics, tensorboard, etc